frontend/packages/db-structure/biome.jsonc

```diff
@@ -8,6 +8,7 @@
   "linter": {
     "rules": {
       "correctness": {
+        "noNodejsModules": "error",
```

Comment:
The `frontend/packages/db-structure/src/schema/` directory (not the parser) likely still requires `noNodejsModules`, I suppose.